projects/word-weaver/src/index.ghpages.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base href="./" />
+    <title>WordWeaver</title>
+
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta property="og:title" content="WordWeaver" />
+    <meta
+      property="og:description"
+      content="Visualization for verb conjugators"
+    />
+    <meta property="og:url" content="https://github.com/roedoejet/wordweaver" />
+
+    <link rel="icon" type="image/png" href="./assets/logo.png" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Karla|Lato|Material+Icons"
+    />
+
+    <!-- asset links -->
+    <link rel="manifest" href="./manifest.webmanifest" />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="./assets/icons/apple-icon-180.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="167x167"
+      href="./assets/icons/apple-icon-167.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="152x152"
+      href="./assets/icons/apple-icon-152.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="120x120"
+      href="./assets/icons/apple-icon-120.png"
+    />
+
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+
+    <link
```

Comment:
This code is just craziness. Do we really need this many different sizes of the same image?  I guess this is not a change in this PR, you just duplicated index.html (why?) but bleh.